partitioned-raft-kv.md

```diff
@@ -37,6 +37,8 @@ v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例
 
 * 暂不支持基于 EBS 的快照备份
 * 暂不支持 Online Unsafe Recovery 和 Titan
-* 暂不支持 tikv-ctl 命令行管理工具
+* 不支持 tikv-ctl 命令行管理工具中的以下子命令：
```

Comment:
the unsafe opeartions that tikv-ctl v2 does not support includes: remove-fail-stores, recreate-region, and reset-to-version.